backend/app.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import FastAPI
+from pymongo import MongoClient
+from dotenv import load_dotenv
+import os
+from api import auth
+
+load_dotenv()
+
+app = FastAPI(
+    title="Qreate",
+    description="FAQ page generator.",
+    version="0.1.0"
+)
+
+MONGO_URL = os.getenv('M_URI')
+
+@app.on_event("startup")
+def startup_db_client():
```

Comment:
For the APIs, we're using `get_db()` function, but here we're attaching the `app` object with database client. This could be changed.